packages/main/src/Slider.js

```diff
@@ -0,0 +1,292 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import SliderTemplate from "./generated/templates/SliderTemplate.lit.js";
+
+// Styles
+import SliderCss from "./generated/themes/Slider.css.js";
+import ScrollEnablement from "@ui5/webcomponents-base/dist/delegate/ScrollEnablement.js";
+
+/**
+ * @private
+ */
+const metadata = {
+	tag: "ui5-slider",
+	properties: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+		/**
+		 * Defines whether the <code>ui5-slider</code> is disabled
+		 * (default is set to <code>false</code>).
+		 * A disabled <code>ui5-slider</code> can't be pressed or
+		 * focused, and it is not in the tab chain.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		disabled: {
+			type: Boolean
+		},
+
+		/**
+		 * Defines the currently selected value
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		value: {
+			type: String
+		},
+
+		/**
+		 * Indicates if the slider will support cyclic scrolling.
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		isCyclic: {
+			type: Boolean
+		},
+
+		/**
+		 * Defines the label of the slider.
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		label: {
+			type: String,
+			defaultValue: ""
+		},
+
+		/**
+		 * Indicates if the slider is expanded.
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		expanded: {
+			type: Boolean
+		},
+
+		_items: {
+			type: Object
+		}
+	},
+	slots: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+
+	},
+	events: /** @lends sap.ui.webcomponents.main.Slider.prototype */ {
+		/**
+		 * Fires when the slider is expanded.
+		 */
+		expanded: {},
+
+		/**
+		 * Fires when the slider is collapsed.
+		 */
+		collapsed: {},
+
+		/**
+		 *  Fires when new value is selected.
+		 */
+		valueSelect: {
+			value: {
+				type: String
+			}
+		}
+	},
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ *
+ * <h3>Usage</h3>
+ *
+ * For the <code>ui5-slider</code>
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import @ui5/webcomponents/dist/Slider.js";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webcomponents.main.Slider
+ * @extends UI5Element
+ * @tagname ui5-slider
+ * @public
+ */
+class Slider extends UI5Element {
+	static get metadata() {
+		return metadata;
+	}
+
+	static get render() {
+		return litRender;
+	}
+
+	static get styles() {
+		return SliderCss;
+	}
+
+	static get template() {
+		return SliderTemplate;
+	}
+
+	static async define(...params) {
+		super.define(...params);
+	}
+
+	constructor() {
+		super();
+
+		this._items = ["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16"];
+		this.value = this._items[0];
```

Comment:
You should not modify public API (value). Use only what the user provided and only change it upon user interaction.